tests/integration/test_model.py

```diff
@@ -663,20 +678,22 @@ def test_mlflow_export_xgboost(c, client, training_df, tmpdir):
             temporary_dir
         )
     )
+
     assert (
         mlflow.sklearn.load_model(str(temporary_dir)).__class__.__name__
         == "DaskXGBClassifier"
     )
 
 
-def test_mlflow_export_lightgbm(c, training_df, tmpdir):
+def test_mlflow_export_lightgbm(c, tmpdir):
     # Test only when mlflow & lightgbm was installed
     mlflow = pytest.importorskip("mlflow", reason="mlflow not installed")
     lightgbm = pytest.importorskip("lightgbm", reason="lightgbm not installed")
+
     c.sql(
         """
         CREATE MODEL IF NOT EXISTS my_model_lightgbm WITH (
-            model_class = 'lightgbm.LGBMClassifier',
+            model_class = 'LGBMClassifier',
```

Comment:
Do we know what happens if our input df's are gpu tables and we try using LightGBM. I don't recall if we error or not when passing a dask-cudf input to lgbm

Reply:
I remember trying it with `LGBMClassifier` and it failed. I'm not sure about all of the others like `LGBMRegressor`, etc. At least with XGBoost, the Regressors worked but the Classifiers failed, so it might be similar behavior.

I have https://github.com/dask-contrib/dask-sql/issues/943 open to investigate some cases where the GPU version fails with `experiment_class`, as many of them fail with the same `TypeError: Implicit conversion to a NumPy array is not allowed`. I could also open an issue to further investigate GPU LightGBM and XGBoost with `model_class`.

Reply:
Opened https://github.com/dask-contrib/dask-sql/issues/1020